src/app/public/testing/colorpicker-fixture.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Allows interaction with a SKY UX colorpicker component.
+ */
+export class SkyColorpickerFixture {
+
+  private debugEl: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(this.fixture, this.skyTestId, 'sky-colorpicker');
+  }
+
+  /**
+   * The colorpicker's currently selected color formatted to the `outputFormat`.
+   */
+  public get value(): string {
+    return this.getColorpickerInputEl().nativeElement.value;
+  }
+
+  /**
+   * Set the colorpicker's color hex code.
+   * @param hexValue The new color hex code. Must inculde '#'.
+   */
+  public async setColorFromHexValue(hexValue: string): Promise<any> {
```

Comment:
Since you have a `get value` property, what about renaming these to `setValueFromHex`, `setValueFromRGBA`, etc.?